spec/requests/users_controller_spec.rb

```diff
@@ -19,6 +19,12 @@
         end.to change(User, :count).by(1)
       end
 
+      it 'enqueues email to be delivered later' do
+        assert_enqueued_jobs 1 do
```

Comment:
Could we change this to use https://relishapp.com/rspec/rspec-rails/v/3-5/docs/matchers/have-been-enqueued-matcher#checking-passed-arguments-to-job